frontend/public/components/secrets/create-secret/SecretFormTitle.tsx

```diff
@@ -0,0 +1,30 @@
+import { useTranslation } from 'react-i18next';
+import { SecretTypeAbstraction } from '.';
+import * as React from 'react';
+
+type SecretFormTitleProps = {
+  isCreate: boolean;
+  typeAbstraction: SecretTypeAbstraction;
+};
+
+export const SecretFormTitle: React.FC<SecretFormTitleProps> = ({ isCreate, typeAbstraction }) => {
+  const { t } = useTranslation();
+  switch (typeAbstraction) {
+    case 'generic':
```

Comment:
Use enum constant instead of "magic strings":
```suggestion
    case SecretTypeAbstraction.generic:
```